servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpSetCookie.java

```diff
@@ -34,13 +34,13 @@
  * Default implementation of {@link HttpSetCookie}.
  */
 public final class DefaultHttpSetCookie implements HttpSetCookie {
-    private static final String ENCODED_LABEL_DOMAIN = "; domain=";
-    private static final String ENCODED_LABEL_PATH = "; path=";
-    private static final String ENCODED_LABEL_EXPIRES = "; expires=";
-    private static final String ENCODED_LABEL_MAX_AGE = "; max-age=";
-    private static final String ENCODED_LABEL_HTTP_ONLY = "; httponly";
-    private static final String ENCODED_LABEL_SECURE = "; secure";
-    private static final String ENCODED_LABEL_SAMESITE = "; samesite=";
+    private static final String ENCODED_LABEL_DOMAIN = "; Domain=";
+    private static final String ENCODED_LABEL_PATH = "; Path=";
+    private static final String ENCODED_LABEL_EXPIRES = "; Expires=";
+    private static final String ENCODED_LABEL_MAX_AGE = "; Max-Age=";
+    private static final String ENCODED_LABEL_HTTP_ONLY = "; HttpOnly";
```

Comment:
The spec doesn't have `MUST` in its wording and clarifies that attribute names are case insensitive:

> NOTE: The name of an attribute-value pair is not case sensitive. So while they are presented here in CamelCase, such as "HttpOnly" or "SameSite", any case is accepted. E.x.: "httponly", "Httponly", "hTTPoNLY", etc.

I don't mind either way. Netty uses CamelCase: https://github.com/netty/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/cookie/CookieHeaderNames.java#L19-L33
We can align with spec and Netty.
Note that it has discrepancy for `HTTPOnly`.